src/components/pages/VendingMachine/MenuSelect.jsx

```diff
@@ -48,6 +50,12 @@ const MenuSelect = () => {
   const availableMilk =
     MENU_PRICE.milk <= coin && milk > 0 && cup > 0 ? false : true;
 
```

Comment:
```suggestion
  const isDiabledMilk =
    MENU_PRICE.milk > coin || milk <= 0 || cup <= 0;

```

availableMilk 변수명이라 가능할 때 조건인 줄 알았는데, 밑에보니까 disabled에 그대로 들어가네요!! 헷갈리지 않게 isDisabled... 나 다른 걸로 변수명을 바꿨으면 좋겠어요! 다른 avaliable~ 변수명도요!